### DISCUSSION TITLE
Any way to remap a mouse button to another mouse button?

### DISCUSSION THREAD

**Discussion by imperfectlink** [sezanzeb/input-remapper#428] [Q&A] [answered]
Any way to remap a mouse button to another mouse button?

It seems as though the only thing it does currently is map a keyboard key to another or a mouse click to a keyboard keystroke but not mouse to mouse. This seems like it would be an obvious use case.

Also it's not immediately obvious that there is even a destination field. Just the unlabeled input field. It might be a good idea to have defined REMAP and TO sections.

**Comment by jonasBoss** [comment, marked as answer]
There is a dropdown menue where you can select the target.
![image](https://user-images.githubusercontent.com/31804124/178815700-1a485d03-d6c3-43b1-a1de-db827f090835.png)

**Reply by imperfectlink** [reply]
Thanks for that. It was the text (BTN_MIDDLE) that wasn't obvious what to do. Is there a sheet of all the options somewhere?

**Reply by jonasBoss** [reply]
The best thing is to use the Autocompletion to see which keys and buttons are available on each device. They should be mostly self explanatory `BTN_MIDDLE` is the middle mouse button (press down scroll wheel) on your mouse. The [documentation](https://github.com/sezanzeb/input-remapper/blob/main/readme/usage.md#key-names) describes how you can get a list of available keycodes. 
Under the hood inputremapper uses the event-codes from the [linux kernel](https://github.com/torvalds/linux/blob/master/include/uapi/linux/input-event-codes.h) each device has a subset of those.
